package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "slash-bot",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Template for slash bot discord",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KurokuTetsuya/slash-bot"
+  },
+  "license": "AGPL-3.0",
+  "author": "KurokuTetsuya <perumnas.good@gmail.com>",
+  "contributors": [
+    "Hazmi35 <contact@hzmi.xyz> (https://hzmi.xyz/)"
```

Comment:
This should be `USERNAME <EMAIL> (HOMEPAGE / WEBSITE)`

Reply:
is homepage really needed?

Reply:
nope, only "repository" is fine